website/source/api/secret/transit/index.html.md

```diff
@@ -747,13 +747,26 @@ be used.
 
 - `input` `(string: <required>)` – Specifies the **base64 encoded** input data.
```

Comment:
This parameter is no longer <required>.

Reply:
I have changed "<required>" to nil - and added a sentence about either input or batch_input being needed.   I'm not sure about "nil" being correct .